internal/encrypt/encrypt.go

```diff
@@ -88,13 +88,20 @@ func (e *Encrypter) Encrypt(reader io.Reader, filename string) error {
 	return nil
 }
 
+const outDirName = "out"
+
 func (e *Encrypter) generateAndSaveMasterKey(filename string) ([]byte, error) {
 	masterKey, err := sss.GenerateMasterKey()
 	if err != nil {
 		return nil, err
 	}
 
-	err = file.WriteKey(masterKey, "out/"+filename)
+	if err := os.MkdirAll(outDirName, 0744); err != nil {
+		fmt.Println(err)
```

Comment:
I would drop the log, we are already returning the error and logging it before quitting (https://github.com/enrichman/stegosecrets/blob/main/cmd/stego/stego.go#L13)

Reply:
Sorry, the log was just to test code. I will remove it.